.gitignore

```diff
@@ -0,0 +1,8 @@
+# Android Studio
```

Comment:
we already have a gitignore file. 
Remove the .idea folder. Only then, will merge this PR